### PR TITLE
feat(ELB): elb pool add params

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  elb pool add params
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  elb pool add params
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/elb/' TESTARGS='-run TestAccElbV3Pool_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/elb/ -v -run TestAccElbV3Pool_basic -timeout 360m -parallel 4
=== RUN   TestAccElbV3Pool_basic
=== PAUSE TestAccElbV3Pool_basic
=== RUN   TestAccElbV3Pool_basic_with_loadbalancer
=== PAUSE TestAccElbV3Pool_basic_with_loadbalancer
=== RUN   TestAccElbV3Pool_basic_with_listener
=== PAUSE TestAccElbV3Pool_basic_with_listener
=== RUN   TestAccElbV3Pool_basic_with_type_ip
=== PAUSE TestAccElbV3Pool_basic_with_type_ip
=== CONT  TestAccElbV3Pool_basic
=== CONT  TestAccElbV3Pool_basic_with_listener
=== CONT  TestAccElbV3Pool_basic_with_type_ip
=== CONT  TestAccElbV3Pool_basic_with_loadbalancer
--- PASS: TestAccElbV3Pool_basic_with_type_ip (69.11s)
--- PASS: TestAccElbV3Pool_basic (71.38s)
--- PASS: TestAccElbV3Pool_basic_with_listener (93.43s)
--- PASS: TestAccElbV3Pool_basic_with_loadbalancer (114.45s) 
PASS 
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/elb       114.517s
```

### CODE REVIEW

File: docs/resources/elb_pool.md [hunk 1]
```diff
@@ -8,14 +8,74 @@ Manages an ELB pool resource within HuaweiCloud.
 
 ## Example Usage
 
+### Created by LoadBalancer ID
```
Comment (ShiChangkuo) [inline docs/resources/elb_pool.md]:
```suggestion
### Create a Pool and Associate a Load Balancer
```

File: docs/resources/elb_pool.md [hunk 2]
```diff
@@ -8,14 +8,74 @@ Manages an ELB pool resource within HuaweiCloud.
...
+  protection_reason = "test reason"
+
+  persistence {
+    type        = "APP_COOKIE"
+    cookie_name = "testCookie"
+  }
+}
+```
+
+### Created by Listener ID
```
Comment (ShiChangkuo) [inline docs/resources/elb_pool.md]:
```suggestion
### Create a Pool and Associate a Listener
```

File: docs/resources/elb_pool.md [hunk 3]
```diff
@@ -8,14 +8,74 @@ Manages an ELB pool resource within HuaweiCloud.
...
+  protection_reason = "test reason"
+
+  persistence {
+    type        = "APP_COOKIE"
+    cookie_name = "testCookie"
+  }
+}
+```
+
+### Created by type
```
Comment (ShiChangkuo) [inline docs/resources/elb_pool.md]:
```suggestion
### Create a Pool and Associate later
```

File: huaweicloud/services/elb/resource_huaweicloud_elb_pool.go [hunk 4]
```diff
@@ -118,6 +99,49 @@ func ResourcePoolV3() *schema.Resource {
...
 			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"type": {
```
Comment (ShiChangkuo) [inline huaweicloud/services/elb/resource_huaweicloud_elb_pool.go]: it's better to put this parameter together with loadbalancer_id and listener_id.

File: docs/resources/elb_pool.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/elb_pool.md]: done

File: docs/resources/elb_pool.md (hunk 2 above)
Comment (houpeng80) [reply docs/resources/elb_pool.md]: done

File: docs/resources/elb_pool.md (hunk 3 above)
Comment (houpeng80) [reply docs/resources/elb_pool.md]: done

File: huaweicloud/services/elb/resource_huaweicloud_elb_pool.go (hunk 4 above)
Comment (houpeng80) [reply huaweicloud/services/elb/resource_huaweicloud_elb_pool.go]: done

**Pull request merged**